mono/mini/interp/interp.c

```diff
@@ -992,7 +991,7 @@ ves_array_calculate_index (MonoArray *ao, stackval *sp, InterpFrame *frame, gboo
 	return pos;
 }
 
-static void
+static MONO_NEVER_INLINE void
```

Comment:
These might merit
comments — “to avoid growing caller’s frame” or somesuch.
NT does similar.